src/absil/il.fs

```diff
@@ -320,7 +320,16 @@ let sha1HashInt64 s = SHA1.sha1HashInt64 s
 // 
 // -------------------------------------------------------------------- 
 
-type ILVersionInfo = uint16 * uint16 * uint16 * uint16
+[<Struct>]
```

Comment:
❓ Can you apply `[StructLayout]` to set the `Pack` to 2?

Reply:
The struct already comes out to 8 bytes, what would `Pack` do in this case?

Reply:
@TIHan seems it doesn't need it then thanks!